includes/class-wc-payments-checkout.php

```diff
@@ -230,10 +230,6 @@ public function get_payment_fields_js_config() {
 		}
 		$payment_fields['enabledBillingFields'] = $enabled_billing_fields;
 
-		if ( WC_Payments::woopay_tracker()->should_enable_tracking() ) {
-			$payment_fields['tracksUserIdentity'] = WC_Payments::woopay_tracker()->tracks_get_identity( get_current_user_id() );
-		}
-
```

Comment:
This was the main culprit. This code is run during page load. `tracks_get_identity` will set a `tk_ai` cookie if it's not already present.